source/javascripts/services/_appService.js.erb

```diff
@@ -209,7 +209,7 @@
 				appService.appConfig.trigger_map = updatedTriggerMap;
 			};
 
-			appService.saveAppConfig = function(userSavedYmlToRepositoryPromise, requestConfig) {
+			appService.saveAppConfig = function(userSavedYmlToRepositoryPromise, requestConfig, tabOpenDuringSave) {
```

Comment:
Please reorder the input args, the `requestConfig` should always be the last one, as it's the least likely defined on calls. The method is not called anywhere else aside from MainController, so there shouldn't be any issues with having a different 2nd arg as before.